h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -58,12 +56,27 @@
   public final GenModel m;
   private final HashMap<String, Integer> modelColumnNameToIndexMap;
   public final HashMap<Integer, HashMap<String, Integer>> domainMap;
+  private final ErrorConsumer errorConsumer;
 
   private final boolean convertUnknownCategoricalLevelsToNa;
   private final boolean convertInvalidNumbersToNa;
   private final boolean useExtendedOutput;
   private final ConcurrentHashMap<String,AtomicLong> unknownCategoricalLevelsSeenPerColumn;
 
+  /**
+   * Observer interface with methods corresponding to errors during the prediction.
+   */
+  public interface ErrorConsumer {
+    /**
+     * Observe transformation error for data from the predicted dataset.
+     *
+     * @param columnName Name of the column for which the error is raised
+     * @param value      Given value that could not be transformed properly
```

Comment:
The original value that could not be transformed properly is returned. This should be desired behavior from my POV.

Reply:
Thinking about it more... would it be better to make an abstract class? Then we can add default implementations for new methods and still stay compatible with older javas.

Reply:
This is a good idea for future. No need to depend on Java8. Done this way.